php/libraries/NDB_BVL_Battery.class.inc

```diff
@@ -478,6 +478,10 @@ class NDB_BVL_Battery extends PEAR
             $query .= " AND b.CenterID IS NULL";
         }
 
+        if ($firstVisit === false) {
+            $query .= " AND (b.firstVisit IS NULL OR b.firstVisit='N')";
+        }
```

Comment:
The definition of "firstVisit='N'" isn't clearly defined. The tests that I wrote (to match what the function doc said the behaviour was) failed because of this not being included. It's a bug, but I don't think anyone uses firstVisit=N, so it's never come up..